src/sys/unix/epoll.rs

```diff
@@ -1,44 +1,34 @@
-use crate::sys::unix::cvt;
-use crate::{Interests, Token};
-
-use libc::{EPOLLET, EPOLLIN, EPOLLOUT};
-use std::os::unix::io::AsRawFd;
-use std::os::unix::io::RawFd;
+use std::os::unix::io::{AsRawFd, RawFd};
 #[cfg(debug_assertions)]
 use std::sync::atomic::{AtomicUsize, Ordering};
 use std::time::Duration;
-use std::{cmp, i32, io};
+use std::{cmp, i32, io, ptr};
+
+use libc::{EPOLLET, EPOLLIN, EPOLLOUT};
+use log::error;
+
+use crate::{Interests, Token};
 
-/// Each Selector has a globally unique(ish) ID associated with it. This ID
-/// gets tracked by `TcpStream`, `TcpListener`, etc... when they are first
-/// registered with the `Selector`. If a type that is previously associated with
-/// a `Selector` attempts to register itself with a different `Selector`, the
-/// operation will return with an error. This matches windows behavior.
+/// Unique id for use as `SelectorId`.
 #[cfg(debug_assertions)]
-static NEXT_ID: AtomicUsize = AtomicUsize::new(0);
+static NEXT_ID: AtomicUsize = AtomicUsize::new(1);
```

Comment:
What is the reason to bump to 1?

Reply:
So we can remove the + 1 in `Selector::new`.

Reply:
👍